docs/catalog/cloud/Use energy efficient hardware.md

```diff
@@ -0,0 +1,42 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- compute
+- kubernetes
+- role:software-engineer
+- role:cloud-engineer
+- size:medium
+---
+
+# Use energy efficient hardware
+
+## Description
+
+Cloud virtual machines are offered with different capabilties, based different hardware processors.
+
+Hence using virtual machines based on efficient processors would impact hardware efficiency and reduce carbon emissions.
+
+
+## Solution
+
+Use new cloud virtual machines that have been engineered with "cloud native processors" to efficiently run scale-out, cloud-native workloads and consume energy efficiently.
+
+
+## SCI Impact
+`SCI = (E * I) + M per R`
+[Software Carbon Intensity Spec](https://grnsft.org/sci)
+
+Regarding the SCI equation, using energy efficient hardware will impact:
+
+- `M`: By using energy efficient hardware, we reduce the total embodied carbon emissions.
```

Comment:
If it is energy efficient, than it would reduce the energy consumption and would be `E`, not `M`, right?

Reply:
indeed !